services/QuillLMS/client/app/bundles/Shared/styles/snackbar.scss

```diff
@@ -7,7 +7,7 @@
   text-align: center;
   padding: 13px 16px;
   position: fixed;
-  z-index: 1;
+  z-index: 9999;
```

Comment:
The snackbar was being hidden behind the Clusters Table.  Not sure if there's a better way to handle this.

Reply:
This seems fine to me. I don't think we'd ever want our snackbar below a different element.